library/templates/address_list_display.php

```diff
@@ -26,12 +26,12 @@
 
 <div id="<?php echo attr($table_id); ?>" class="row mt-3">
     <div class="table_text_addresses col-12">
-        <div class ="text_addresses_header label_custom pl-1 bg-gray300" style="text-align: left !important; padding-top: 0.1rem;">
+        <div class ="text_addresses_header label_custom pl-1" style="text-align: left !important; padding-top: 0.1rem;; background-color: var(--gray300)">
```

Comment:
This works for me I guess. Still would have preferred setting a class in navmenu.scss(or core theme) as this is a useful shade.